cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfJointComponent.h

```diff
@@ -0,0 +1,27 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#pragma once
+
+#include <CoreMinimal.h>
+#include <PhysicsEngine/PhysicsConstraintComponent.h>
+
+#include "UrdfJointComponent.generated.h"
+
+class UUrdfLinkComponent;
```

Comment:
Classes first, then enums, then structs, with white space in between (for consistency with our other code).

```
class UUrdfLinkComponent;

enum class UrdfJointType;

struct UrdfJointDesc;
```

Reply:
fixed